packages/augur-sdk/src/state/logs/types.ts

```diff
@@ -200,6 +200,13 @@ export enum OrderEventType {
   Fill = 3,
 }
 
+export enum OrderState {
```

Comment:
trading.ts has these OrderState values as well. are they needed here?

Reply:
Are you referring to the order_type defined in constants? If so, those are different things.